### PR TITLE
Log4jconfig

### PR DESCRIPTION
**Pull request by budaqing**
add node['kafka-cluster']['config']['log4j'] attribute so that it's easier to customize log4j config of kafka.

usage:

node.default['kafka-cluster']['config']['log4j']['customized'] = true
node.default['kafka-cluster']['config']['log4j']['maxFileSize'] = "10MB"
node.default['kafka-cluster']['config']['log4j']['maxNumFiles'] = '10'
node.default['kafka-cluster']['config']['log4j']['level']['kafka'] = "DEBUG"
include_recipe 'kafka-cluster::default'

### CODE REVIEW

**Comment by budaqing** [comment]
@johnbellone 
Ruby 2.2 check passed but Ruby 2.1 failed. I don't understand why?

File: libraries/kafka_config.rb [hunk 1]
```diff
@@ -44,6 +62,32 @@ def to_s
             content new_resource.to_s
             mode '0644'
           end
+
+          log "#{new_resource.log4j}" do
```
Comment (johnbellone) [inline libraries/kafka_config.rb]: Would you mind removing this?

**Comment by johnbellone** [comment]
After the above :+1: for merge.

**Comment by johnbellone** [comment]
I can remove it.

**Pull request merged**